tests/integration/http-verbs-test.js

```diff
@@ -99,6 +103,20 @@ test('mirage responds to patch', function(assert) {
   });
 });
 
+test('mirage responds to resource', function(assert) {
+  assert.expect(0);
+  let done = assert.async();
+
+  this.server.resource('/contacts');
```

Comment:
I think the api should be `this.resource('contacts')` - you're declaring a contact, not defining a url